app/pages/project/components/ProjectNavbar/helpers/getProjectLinks.js

```diff
@@ -46,8 +46,8 @@ function getProjectLinks({ project, projectRoles, workflow, user }) {
 
   // For projects with external front ends
   if (redirect) {
-    const newUrl = redirect.replace(/\/?#?\/+$/, '');
-    links.classify.url = `${newUrl}/#/classify`;
+    const redirectUrl = `${redirect.replace(/\/+$/, '')}/classify`;
```

Comment:
👍 Nice, this will catch redirect URLs that either end or don't end with slashes (or multiple slashes).